sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3459,6 +3459,16 @@ object SQLConf {
       .checkValues(Set("legacy", "row", "dict"))
       .createWithDefaultString("legacy")
 
+  val PYSPARK_HIDE_TRACEBACK =
+    buildConf("spark.sql.execution.pyspark.udf.hideTraceback.enabled")
+      .doc(
+        "When true, only show the message of the exception from Python UDFs, " +
+        "hiding stack trace and exception type."
+      )
```

Comment:
nit
```suggestion
        "hiding stack trace and exception type.")
```

Reply:
I think we should only hide stack trace but not the Python error class (exception type)?

Reply:
Right, I forgot to update the description after updating code. Thanks for pointing out.